docs/encyclopedia/retry-policies.mdx

```diff
@@ -101,13 +103,151 @@ Non-Retryable Errors = []
 - **Use case:** Use this attribute to ensure that retries do not continue indefinitely.
   In most cases, we recommend using the Workflow Execution Timeout for [Workflows](/workflows) or the Schedule-To-Close Timeout for Activities to limit the total duration of retries, rather than using this attribute.
 
-### Non-Retryable Errors
+### Non-Retryable Errors {#non-retryable-errors}
```

Comment:
Just something I've been wondering about. Why do we do the `{#....}` on some of the headings?